### DISCUSSION TITLE
Is it possible to disable horizontal scrolling?

### DISCUSSION THREAD

**Discussion by mollerhoj** [nvim-tree/nvim-tree.lua#2053] [Q&A]
When scrolling with my trackpad, I usually end up scrolling horizontally, which makes it impossible to read the filenames:

![Screenshot 2023-03-13 at 09 56 15](https://user-images.githubusercontent.com/649188/224653201-8f1b4641-5570-4c23-8528-bbbad2aa8e51.png)

**Comment by alex-courtis** [comment]
That's an nvim setting that's beyond the control of nvim-tree.

See `:help mousescroll` to change.

**Reply by mollerhoj** [reply]
Thanks, `vim.api.nvim_exec('set mousescroll=hor:0', true)` did the trick :-)
